.github/workflows/ci.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
       - name: Build
         run: ${{ matrix.build_cmd }}
       - name: Run Selftest
-        if: ${{ matrix.target.arch != 'macos12-arm64' }}
+        if: ${{ matrix.target.arch != 'macos14-arm64' }}
```

Comment:
I don't understand, why we skip selftest for macos-arm64. Looks like it should work. Please check

Reply:
Removed the condition to skip the sleftest for macos-arm64 target. Now selftest is running on arm64 based targets as well.